ferrocene/doc/user-manual/src/self-test/error-codes.rst

```diff
@@ -72,15 +71,15 @@ FST_007: Binary version mismatch
 --------------------------------
 
 This error occurs when the version of the ``ferrocene-self-test`` binary and
-that of the Ferrocene compiler are not the same.
+that of rustc are not the same.
 
 **Suggested fixes**
 
 Do not use the same ``ferrocene-self-test`` binary with different Ferrocene
 toolchains.
 
 Reinstall the Ferrocene toolchain in order to obtain the expected
-``ferrocene-self-test`` binary or Ferrocene compiler.
+``ferrocene-self-test`` binary or rustc.
```

Comment:
referring to a command/executable, so feels better to quote it (and also for consistency, since ferrocene-self-test command is also quoted)
```suggestion
``ferrocene-self-test`` binary or ``rustc``.
```